includes/class-sensei-modules.php

```diff
@@ -1243,28 +1243,69 @@ public function course_columns( $columns = array() ) {
 	 */
 	public function course_column_content( $column = '', $course_id = 0 ) {
 		if ( 'modules' === $column ) {
-			$modules = $this->get_course_modules( $course_id );
-
-			if ( $modules ) {
-				$module_links = [];
-				foreach ( $modules as $module ) {
-					$module_links[] = sprintf(
-						'<a href="%s">%s</a>',
-						esc_attr( get_edit_term_link( $module->term_id ) ),
-						esc_html( $module->name )
-					);
-				}
+			$this->output_course_modules_column( $course_id );
+		}
+	}
 
-				echo implode( ', ', $module_links ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output is escaped in the array.
+	/**
+	 * Output the course modules column HTML.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param int $course_id
+	 * @return void
```

Comment:
`@return void` is not needed (I mentioned this in another PR but am too lazy to track it down 😆 ).

Reply:
Voided the void in 24a3f35e9cd656f3e9e417e2db2047257ec04f1a. 🙂